src/taco-cli/cli/run.ts

```diff
@@ -160,17 +160,12 @@ class Run extends commands.TacoCommandBase {
         var telemetryProperties: tacoUtility.ICommandTelemetryProperties = {};
         return Q.all<any>([Settings.determinePlatform(commandData), Settings.loadSettingsOrReturnEmpty()])
             .spread((platforms: Settings.IPlatformWithLocation[], settings: Settings.ISettings) => {
-            buildTelemetryHelper.storePlatforms(telemetryProperties, "actuallyBuilt", platforms, settings);
-            return Q.all(platforms.map((platform: Settings.IPlatformWithLocation): Q.Promise<any> => {
-                switch (platform.location) {
-                    case Settings.BuildLocationType.Local:
-                        // Just run local, and failures are failures
-                        return CordovaWrapper.run(commandData, platform.platform);
-                    case Settings.BuildLocationType.Remote:
-                        // Just run remote, and failures are failures
-                        return Run.runRemotePlatform(platform.platform, commandData, telemetryProperties);
-                }
-            }));
+                buildTelemetryHelper.storePlatforms(telemetryProperties, "actuallyBuilt", platforms, settings);
+
+                return Settings.operateOnPlatforms(platforms,
```

Comment:
We are trying to move towards more explicit typings, so that's why I've added them here. Just spoke with @nisheetjain and he agrees that even though it isn't currently complained about by tslint, it's worth adding just to be sure.